packages/table/src/Row/InternalRowWithRT.tsx

```diff
@@ -96,6 +96,7 @@ const InternalRowWithRT = <T extends LGRowData>({
         )}
         data-selected={isSelected}
         aria-hidden={!isRowVisible}
+        data-expanded={isExpanded}
```

Comment:
do we need this?

Reply:
Yes. This data attribute is the only way i can check if a row is expanded in the DOM.